public/api/private/notes/old

```diff
@@ -1,13 +1,18 @@
 {
```

Comment:
Shouldn't this file be named `old-get`?

Reply:
do we even need this file?

Reply:
Well, the intention of this old note was to test whether the redirect from `/old` to `/n/old` works (cypress test missing btw [but not in this PR]).

Reply:
Well we should be able to test the redirector without a real backend.

Reply:
fixed